### PR TITLE
README edits (2)

### PR DESCRIPTION
**Pull request by Phoebe-B**
Edits to the following README sections:
• Customising the SDK

Improving structure and language (no content changes).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - [Getting started](#getting-started)
 - [Handling callbacks](#handling-callbacks)
 - [Removing the SDK](#removing-the-sdk)
+- [Initilization options](#initilization-options)
```
Comment (superdevx64) [inline README.md:12]: Minor: "Initialization" (both times)

File: README.md [hunk 2]
```diff
@@ -277,9 +278,7 @@ onfidoOut = Onfido.init({...})
 onfidoOut.tearDown()
 ```
 
-## Customising the SDK
-
-A number of options are available to allow you to customize the SDK:
+## Initilization options
```
Comment (superdevx64) [inline README.md:281]: Minor: "Initialization"

**Comment by superdevx64** [comment]
General but minor comment which may be out of scope: a few words use British spelling rather than Oxford spelling (ise rather than ize)

File: README.md [hunk 3]
```diff
@@ -321,13 +318,13 @@ A number of options are available to allow you to customize the SDK:
...
-  To change the state of the modal after calling `init()` you need to later use `setOptions()` to modify it.
-  The default value is `false`.
+  Defines whether the modal is open or closed, if `useModal` is set to `true`. The default value is `false`.
+
+  To change the state of the modal after calling `init()` you need to use `setOptions()`.
 
 - **`shouldCloseOnOverlayClick {Boolean} optional`**
 
-  In case `useModal` is set to `true`, the user by default can close the SDK by clicking on the close button or on the background overlay. You can disable the user from closing the SDK on background overlay click by setting the `shouldCloseOnOverlayClick` to false.
+  If `useModal` is set to `true`, by default the user can close the SDK by clicking on the close button or on the background overlay. You can disable the user's ability to close the SDK by clicking the background overlay through setting `shouldCloseOnOverlayClick` to false.
```
Comment (kopijunkie) [inline README.md:327]:
```suggestion
  If `useModal` is set to `true`, by default the user can close the SDK by clicking on the close button or on the background overlay. You can disable the user's ability to close the SDK by clicking the background overlay through setting `shouldCloseOnOverlayClick` to `false`.
```

File: README.md [hunk 4]
```diff
@@ -743,103 +756,109 @@ A number of options are available to allow you to customize the SDK:
...
-    ```
+- `forceCrossDevice` (boolean - default: `false`)
 
-  - `useLiveDocumentCapture` (boolean - default: `false`)
-    **This BETA feature is only available on mobile devices.**
+  The Web SDK offers a cross device flow where desktop users will be given the option to continue using their desktop browser or swap to using their mobile device browser to complete the capture process. If a user selects to use their mobile device they will be redirected via a secure link that they can receive by SMS or QR code to complete the flow. At the end of the capture process users will be redirected back to their desktop to complete the SDK flow.
 
-    When set to `true`, users on mobile browsers with camera support will be able to capture document images using an optimised camera UI, where the SDK directly controls the camera feed to ensure live capture. For unsupported scenarios, see the `uploadFallback` section below.
-    Tested on: Android Chrome `78.0.3904.108`, iOS Safari `13`
+  When `forceCrossSevice` is set to `true`, the cross device flow is mandatory for all users. Desktop users will be required to complete the capture process on a mobile device browser.
```
Comment (kopijunkie) [inline README.md:763]:
```suggestion
  When `forceCrossDevice` is set to `true`, the cross device flow is mandatory for all users. Desktop users will be required to complete the capture process on a mobile device browser. 
```

**Review by kopijunkie — CHANGES_REQUESTED** [review]
Just some minor typos, Markdown formatting that needs fixing.

**Review by kopijunkie — APPROVED** [review]

**Pull request merged**